lib/chai/interface/assert.js

```diff
@@ -1012,49 +1012,49 @@ module.exports = function (chai, util) {
 
   /**
    * ### .nestedInclude(haystack, needle, [message])
-   * 
-   * Asserts that 'haystack' includes 'needle'. 
-   * Can be used to assert the inclusion of a subset of properties in an 
+   *
+   * Asserts that 'haystack' includes 'needle'.
+   * Can be used to assert the inclusion of a subset of properties in an
```

Comment:
trailing space are removed by atom i guest. Would this affect documentation? If yes i will carefully add space on `every` next line. So next line will be `*<space><space>...`

Reply:
Well, I would not worry about this that much even though we've already got #1042.

Reply:
Alright, the overall explanation make it clearer. I will process when i had chance :smile: , glad to see you back. :love_letter: